src/SiteRecoveryManagement/Properties/AssemblyInfo.cs

```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.5.1.0")]
+[assembly: AssemblyFileVersion("0.6.0.0")]
```

Comment:
Done. It would be great if the owner of this file could update it though. It auto-updates so I have to explicitly revert it on every build.